packages/gestalt/src/IconButtonFloating.js

```diff
@@ -0,0 +1,69 @@
+// @flow strict
+import { forwardRef, type Node, useImperativeHandle, useRef } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import icons from './icons/index.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
+
+type Props = {|
+  /**
+   * String that clients such as VoiceOver will read to describe the icon button. Always localize the label. See [Accessibility section](https://gestalt.pinterest.systems/web/modalalert#Accessibility) for more info.
+   */
+  accessibilityLabel: string,
+  icon?: $Keys<typeof icons>,
```

Comment:
Is there a default? Should it be required?

I think it should be required. 
![Screen Shot 2022-11-29 at 6 05 26 PM](https://user-images.githubusercontent.com/10593890/204667822-9eba0e90-4e21-46da-a34d-31346b0cacb9.png)